osquery_rules/mac/osquery_mac_unwanted_chrome_extensions.yml

```diff
@@ -0,0 +1,94 @@
+AnalysisType: rule
+Filename: osquery_mac_unwanted_chrome_extensions.py
+PolicyID: Osquery.Mac.UnwantedChromeExtensions
+DisplayName: OSQuery Detected Unwanted Chrome Extensions
+Enabled: true
+ResourceTypes:
+  - Osquery.Differential
+Tags:
+  - Osquery
+  - Panther Enterprise
```

Comment:
This is in the open source analysis repo, should this be tagged `Panther Enterprise`?